src/client/datascience/utils.ts

```diff
@@ -79,42 +77,6 @@ export function translateCellToNative(
     }
 }
 
-export function translateCellFromNative(cell: NotebookCell): ICell {
```

Comment:
I think this is also needed for gather in the interactive window

Reply:
Not sure why we would want to convert a VS Code Notebook cell into ICell for interactive window, it goes the other way. I couldn't find any code.
Please let me know what it is, i couldln't find it.